### PR TITLE
Add codecov badge

### PR DESCRIPTION
**Pull request by feliwir**
Ignore specific folders that should not count into coverage

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ignore:
+    - "deps/**/*"
+    - "tests/**/"
```
Comment (xezon) [inline codecov.yml]: eol

Comment (feliwir) [reply codecov.yml]: Fixed

**Review by xezon — COMMENTED** [review]
Why are yml files removed when this change is about adding codecov widget?

**Comment by feliwir** [comment]
> Why are yml files removed when this change is about adding codecov widget?

Those are artifacts from the past - travis and appveyor are not used anymore. Removing them made sense to me

**Review by xezon — APPROVED** [review]

**Pull request merged**